Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Controllers/RadioNavigationalWarningsAdminControllerTest.cs

```diff
@@ -60,12 +62,44 @@ public void WhenAddRadioNavigationWarningsReturnTrueInRequest_ThenNewRecordIsCre
             Assert.AreEqual("Record created successfully!", _controller.TempData["message"].ToString());
         }
 
+
         [Test]
-        public void WhenICallIndexViewWithParameters_ThenReturnView()
+        public void WhenAddRadioNavigationWarningsReturnFalseInRequest_ThenNewRecordIsNotCreated()
         {
-            A.CallTo(() => _fakeRnwService.GetRadioNavigationWarningsForAdmin(1, 0, null, string.Empty)).Returns(GetFakeRadioNavigationWarningsForAdmin());
-            Task<IActionResult> result = _controller.Index(pageIndex: 1, warningType: 1, year: 2020);
+            var httpContext = new DefaultHttpContext();
+            var tempData = new TempDataDictionary(httpContext, A.Fake<ITempDataProvider>());
```

Comment:
Remove unused variable tempData  and httpContext.

Reply:
done required changes.